applications/RomApplication/custom_utilities/rom_residuals_utility.h

```diff
@@ -138,17 +140,183 @@ namespace Kratos
         return MatrixResiduals;
         }
 
+        Matrix GetProjectedResidualsOntoPsi()
+        {
+            // Getting the number of elements and conditions from the model
+            const int nelements = static_cast<int>(mpModelPart.Elements().size());
+            const int nconditions = static_cast<int>(mpModelPart.Conditions().size());
+
+            const auto& CurrentProcessInfo = mpModelPart.GetProcessInfo();
+            const auto el_begin = mpModelPart.ElementsBegin();
+            const auto cond_begin = mpModelPart.ConditionsBegin();
+
+            //contributions to the system
+            Matrix LHS_Contribution = ZeroMatrix(0, 0);
+            Vector RHS_Contribution = ZeroVector(0);
```

Comment:
```suggestion
            Matrix LHS_Contribution;
            Vector RHS_Contribution;
```
I think you don't need the initialization in this case as you would resize later on.